chalice/deploy/deployer.py

```diff
@@ -169,6 +177,104 @@ def _validate_manage_iam_role(config):
             )
 
 
+class ChaliceDeploymentError(Exception):
+    def __init__(self, error):
+        # type: (Any) -> None
```

Comment:
Prefer to avoid `Any` unless it's really not practical.  This applies to all the `Any`'s in this class.